src/py-opentimelineio/opentimelineio/adapters/cmx_3600.py

```diff
@@ -453,45 +453,46 @@ def make_clip(self, comment_data):
             # variations of EDL, so if we are lenient then maybe we
             # can handle more of them? Only real-world testing will
             # determine this for sure...
-            m = re.match(
-                r'(\d\d:\d\d:\d\d:\d\d)\s+(\w*)\s+(.*)',
-                comment_data["locator"]
-            )
-            if m:
-                marker = schema.Marker()
-                marker.marked_range = opentime.TimeRange(
-                    start_time=opentime.from_timecode(
-                        m.group(1),
-                        self.edl_rate
-                    ),
-                    duration=opentime.RationalTime()
+            for locator in comment_data['locators']:
+                m = re.match(
+                    r'(\d\d:\d\d:\d\d:\d\d)\s+(\w*)\s+(.*)',
+                    locator
                 )
+                if m:
```

Comment:
This is quite a lot of code indented in this block, does the loop work the same if you invert this if-test and dedent the block?  It might help with readability.

```python
if not m:
  continue

marker = schema.Marker()
marker.marked_range...
```

Reply:
Good point, it does make the code a little more readable !
Thanks !